src/modules/Favorites/Favorites.tsx

```diff
@@ -0,0 +1,65 @@
+import firebase from 'firebase'
+import React from 'react'
+
+import { LinkCard } from '../../components/LinkCard'
+import { Collections } from '../../enums/firebaseCollections'
+import { useCurrentUser } from '../../lib/useCurrentUser'
+import type { LinkType } from '../Dashboard'
+
+import {
+    FavoritesListContent,
+    FavoritesListRoot,
+    FavoritesRoot,
+    FavoritesTitle,
+} from './Favorites.styles'
+
+export const Favorites: React.FunctionComponent = () => {
+    const user = useCurrentUser()
+
+    const [links, setLinks] = React.useState<LinkType[]>([])
+
+    const fetchLinks = () => {
+        void firebase
+            .firestore()
+            .collection(Collections.LINKS)
+            .where('userId', '==', user?.id)
```

Comment:
You can add another `where` condition to get only favorite links like so

`.where('isFavourite', '==', true)`

That way you don't have to filter the list in the `map`